addons/mail/static/src/discuss/call/common/call_participant_card.xml

```diff
@@ -31,6 +31,7 @@
                         draggable="false"
                 />
             </div>
+            <button t-if="rootHover.isHover and isContextMenuAvailable and props.minimized" t-on-click="onContextMenu" class="btn btn-secondary btn-sm rounded-circle position-absolute translate-middle o-discuss-CallParticipantCard-contextButton"><i class="fa fa-chevron-down" /></button>
```

Comment:
I would put the chevron down in top-right corner of card.
Move the muted and so on in bottom right.

Also in mobile there's no mouse-hovering: make sure to display the button 100% the time, but it should have nice UI.

I would go with something like this:
<img width="111" alt="Screenshot 2025-01-13 at 17 10 00" src="https://github.com/user-attachments/assets/d820dde4-b680-4a8c-9545-c6daf134c59d" />